integration-tests/tests/utils/aws/dynamoDB.ts

```diff
@@ -1,121 +1,44 @@
-import {
-  DeleteItemCommand,
-  GetItemCommand,
-  PutItemCommand
-} from '@aws-sdk/client-dynamodb'
-
-import { AUDIT_REQUEST_DYNAMODB } from '../../constants/awsParameters'
 import {
   ZendeskFormFieldIDs,
   ZENDESK_END_USER_EMAIL,
   ZENDESK_END_USER_NAME
 } from '../../constants/zendeskParameters'
-import { dynamoDBClient } from './dynamoDBClient'
 import { DynamoDBItem, ItemDetails } from '../../types/dynamoDBItem'
+import { invokeDynamoOperationsLambda } from './invokeDynamoOperationsLambda'
+
+export const getValueFromDynamoDB = async (
+  ticketId: string,
```

Comment:
Is `ticketId` and `zendeskId` being used interchangeably? Perhaps `zendeskId` be preferred and then you can get rid of `ticketId` on line 16?

Reply:
nice spot, I've pushed a change to keep it consistent (zendeskId) in that file